app/js/main.js

```diff
@@ -201,6 +216,40 @@ function setupMeteringDemo(subscriptions) {
     location.reload();
   });
 
+  // Determine whether there is a publisher based entitlement that
+  // unlocks the article. Showcase should be informed if the publisher
+  // unlocks the page due to using their own meter, managing their
+  // own subscription or because the page is free.
+  // This query params check is for easy testing, replace this
+  // with your own code.
+  switch (getQueryParams().publisherEntitlement) {
+    case ('free', ShowcaseEntitlementEvent.FREE):
+      subscriptions.setShowcaseEntitlement({
+        isUserRegistered: null, // Set this if known
```

Comment:
This comment seems vague. Could it say "Set this to a boolean if you know whether the user is registered"?